model/src/test/kotlin/licenses/ResolvedLicenseTest.kt

```diff
@@ -19,41 +19,74 @@
+                        line = 2
+                    ),
+                    matchingPathExcludes = emptyList()
                 ),
-                matchingPathExcludes = emptyList()
-            ),
-            ResolvedCopyrightFinding(
-                statement = "Copyright (C) 2022 The ORT Project Authors",
-                location = TextLocation(
-                    path = "/path/to/file/B",
-                    line = 2
+                ResolvedCopyrightFinding(
+                    statement = "Copyright (C) 2022 The ORT Project Authors",
+                    location = TextLocation(
+                        path = "/path/to/file/B",
+                        line = 2
+                    ),
+                    matchingPathExcludes = emptyList()
                 ),
-                matchingPathExcludes = emptyList()
-            ),
-        )
+            )
 
-        val resolvedCopyrights = originalFindings.toResolvedCopyrights(process = false)
+            val resolvedCopyrights = originalFindings.toResolvedCopyrights(process = false)
 
-        resolvedCopyrights shouldHaveSize 1
-        with(resolvedCopyrights.first()) {
-            statement shouldBe "Copyright (C) 2022 The ORT Project Authors"
-            findings shouldHaveSize 2
-            findings.map { it.location.path }.shouldContainExactlyInAnyOrder("/path/to/file/A", "/path/to/file/B")
+            resolvedCopyrights shouldHaveSize 1
+            with(resolvedCopyrights.first()) {
+                statement shouldBe "Copyright (C) 2022 The ORT Project Authors"
+                findings shouldHaveSize 2
+                findings.map { it.location.path }.shouldContainExactlyInAnyOrder("/path/to/file/A", "/path/to/file/B")
+            }
+        }
+
+        "be consolidated with processing if their statements are afterwards the same" {
+            val originalFindings = listOf(
+                ResolvedCopyrightFinding(
+                    statement = "Copyright (C) 2022 The ORT Project Authors",
+                    location = TextLocation(
+                        path = "/path/to/file/A",
+                        line = 2
+                    ),
+                    matchingPathExcludes = emptyList()
+                ),
+                ResolvedCopyrightFinding(
+                    // Note the "." at the end.
+                    statement = "Copyright (C) 2022 The ORT Project Authors.",
+                    location = TextLocation(
+                        path = "/path/to/file/B",
+                        line = 2
+                    ),
+                    matchingPathExcludes = emptyList()
+                ),
+            )
+
+            val resolvedCopyrights = originalFindings.toResolvedCopyrights(process = true)
+
+            resolvedCopyrights shouldHaveSize 1
+            with(resolvedCopyrights.first()) {
+                statement shouldBe "Copyright (C) 2022 The ORT Project Authors"
+                findings shouldHaveSize 2
```

Comment:
The assertion in line 87 is redundant and should be dropped. See line 88.

Same in line 56.

Reply:
(Same also in previous commit)